opencti-platform/opencti-graphql/src/migrations/1713253755013-convert_linked-to_related-to.js

```diff
@@ -0,0 +1,195 @@
+import { logApp } from '../config/conf';
+import { elDeleteByQueryForMigration, elList, elRawCount, elReindexByQueryForMigration, elUpdateByQueryForMigration } from '../database/engine';
+import {
+  INDEX_STIX_CORE_RELATIONSHIPS,
+  READ_DATA_INDICES,
+  INDEX_STIX_META_RELATIONSHIPS,
+  READ_INDEX_STIX_META_RELATIONSHIPS,
+  READ_RELATIONSHIPS_INDICES,
+  READ_INDEX_STIX_CORE_RELATIONSHIPS
+} from '../database/utils';
+import { RELATION_RELATED_TO } from '../schema/stixCoreRelationship';
+import { executionContext, SYSTEM_USER } from '../utils/access';
+
+export const up = async (next) => {
```

Comment:
please rebase master and rename the migration to update the timestamp, there have been other migrations merged in the meantime.